src/riak_kv_qry_compiler.erl

```diff
@@ -1640,6 +1672,39 @@ basic_select_arith_1_test() ->
        Sel
       ).
 
+varchar_literal_test() ->
+    {ok, Rec} = get_query("SELECT 'hello' from mytab"),
```

Comment:
I note that under normal circumstances, validation of a SELECT query without WHERE clause will not succeed. Is there a plan to allow WHERE-less queries with a static expression not referencing any columns?

(I understand these tests only deal with parts of DDL structure, so the above question is asked in a broader context.)